ui/src/main/java/org/pentaho/di/ui/spoon/Spoon.java

```diff
@@ -4616,7 +4618,7 @@ public void createCmdLineFile() {
         // See if the file already exists...
         int id = SWT.YES;
         try {
-          FileObject f = KettleVFS.getFileObject( filename );
+          FileObject f = KettleVFS.getInstance( DefaultBowl.getInstance() ).getFileObject( filename );
```

Comment:
Instead of DefaultBowl().getInstance(), can it use the managementBowl

Reply:
updated to use executionBowl